tests/ui_tests/test_admin_page.py

```diff
@@ -397,13 +397,41 @@ def test_add_ldap_group_to_users(self, user, users_page, sdk_client_fs, ldap_use
     def test_add_group_to_ldap_users(self, user, users_page, sdk_client_fs, ldap_user_in_group):
         """Check that user can add group to ldap user"""
 
-        users_page.header.wait_success_job_amount_from_header(1)
+        with allure.step("Wait ldap integration ends"):
+            users_page.header.wait_success_job_amount_from_header(1)
         test_group = sdk_client_fs.group_create('test_group')
         users_page.update_user_info(ldap_user_in_group['name'], group=test_group.name)
         with allure.step(f'Check user {user.username} is listed in users list with changed params'):
             user_row = users_page.get_user_row_by_username(ldap_user_in_group['name'])
             assert test_group.name in user_row.text, "User group didn't changed"
 
+    @pytest.mark.ldap()
+    @pytest.mark.usefixtures("configure_adcm_ldap_ad")
+    def test_filter_users(self, user, users_page, sdk_client_fs, ldap_user_in_group):
+        """Check that users can be filtered"""
+
+        with allure.step("Wait ldap integration ends"):
+            users_page.header.wait_success_job_amount_from_header(1)
```

Comment:
Maybe just launch sync action?
It seems faster than to wait ~1min

Reply:
tried wait_for_task_and_assert_result(sdk_client_fs.adcm().action(name="run_ldap_sync").task(), 'success') but it always fails with ObjectNotFound

Reply:
+